hypertrace-view-generator/helm/templates/view-generation/view-generator-deployment.yaml

```diff
@@ -36,6 +36,11 @@ spec:
         - name: log4j-config
           configMap:
             name: {{ $global.Values.logConfig.name }}
+      {{- range $viewgen := $group.generator.serviceConfig.viewgenerators }}
+        - name: {{ $viewgen }}-config
+          configMap:
+            name: {{ $viewgen }}-config
```

Comment:
It seems to me that names are not matching.
configMap - backend-entity-view-generator-config
viewGenName - view-gen-backend-entity-config

Reply:
Done